Enabling debugging on searxng-docker causes server crash

## Issue
Setting debug to **true** prevents the searxng-docker instance from starting properly, which wasn't an issue before the latest commit.

## Config
This is my current configuration, I've commented out the problematic property.
```
# see https://docs.searxng.org/admin/settings/settings.html#settings-use-default-settings
use_default_settings:
  engines:
    remove:
      - bing
      - yep
      - cloudflareai
general:
  #debug: true
  instance_name: "REDACTED"
server:
  # base_url is defined in the SEARXNG_BASE_URL environment variable, see .env and docker-compose.yml
  secret_key: "REDACTED"  # change this!
  limiter: true  # enable this when running the instance for a public usage on the internet
  image_proxy: true
  http_protocol_version: "1.1"
  public_instance: true
ui:
  static_use_hash: true
  theme_args:
    simple_style: black
redis:
  url: redis://redis:6379/0
search:
  safe_search: 0
  autocomplete: "duckduckgo"
  default_lang: "all"
engines:
  - name: startpage
    disabled: false
  - name: yahoo
    disabled: false
  - name: brave
    disabled: true
  - name: mullvadleta brave
    disabled: false
checker:
  scheduling:
```

## Logs
```
Jul 04 11:03:33 searx docker[80054]: searxng  | INFO    searx                         : max_request_timeout=None
Jul 04 11:03:33 searx docker[80054]: searxng  | WARNING searx                         : Be aware you have activated features intended only for public instances. This force the usage of the limiter and link_token / see https://docs.searx>
Jul 04 11:03:34 searx docker[80054]: searxng  | INFO    searx                         : version: 2025.7.4+01be261
Jul 04 11:03:34 searx docker[80054]: searxng  | DEBUG   searx.data                    : init searx.data.ENGINE_DESCRIPTIONS
Jul 04 11:03:34 searx docker[80054]: searxng  | DEBUG   searx.data                    : init searx.data.USER_AGENTS
Jul 04 11:03:34 searx docker[80054]: searxng  | DEBUG   searx.data                    : init searx.data.WIKIDATA_UNITS
Jul 04 11:03:34 searx docker[80054]: searxng  | DEBUG   searx.data                    : init searx.data.EXTERNAL_URLS
Jul 04 11:03:34 searx docker[80054]: searxng  | DEBUG   searx.data                    : init searx.data.ENGINE_TRAITS
Jul 04 11:03:34 searx docker[80054]: searxng  | DEBUG   searx.data                    : init searx.data.EXTERNAL_BANGS
Jul 04 11:03:34 searx docker[80054]: searxng  | DEBUG   searx.webapp                  : static directory is /usr/local/searxng/searx/static
Jul 04 11:03:34 searx docker[80054]: searxng  | DEBUG   searx.webapp                  : templates directory is /usr/local/searxng/searx/templates
Jul 04 11:03:34 searx docker[80054]: searxng  | INFO    searx.webapp                  : in reloading mode and not in main loop, cancel the initialization
Jul 04 11:03:34 searx docker[80054]: searxng  | [INFO] Started worker-1
Jul 04 11:03:34 searx docker[80054]: searxng  | [INFO] Started worker-1 runtime-1
Jul 04 11:03:35 searx docker[80054]: searxng  | [ERROR] Application callable raised an exception
Jul 04 11:03:35 searx docker[80054]: searxng  | Traceback (most recent call last):
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/venv/lib/python3.12/site-packages/granian/wsgi.py", line 51, in _runner
Jul 04 11:03:35 searx docker[80054]: searxng  |     rv = callback(scope, resp)
Jul 04 11:03:35 searx docker[80054]: searxng  |          ^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/venv/lib/python3.12/site-packages/flask/app.py", line 1536, in __call__
Jul 04 11:03:35 searx docker[80054]: searxng  |     return self.wsgi_app(environ, start_response)
Jul 04 11:03:35 searx docker[80054]: searxng  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/searx/flaskfix.py", line 70, in __call__
Jul 04 11:03:35 searx docker[80054]: searxng  |     return self.wsgi_app(environ, start_response)
Jul 04 11:03:35 searx docker[80054]: searxng  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/venv/lib/python3.12/site-packages/werkzeug/middleware/proxy_fix.py", line 183, in __call__
Jul 04 11:03:35 searx docker[80054]: searxng  |     return self.app(environ, start_response)
Jul 04 11:03:35 searx docker[80054]: searxng  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/venv/lib/python3.12/site-packages/flask/app.py", line 1514, in wsgi_app
Jul 04 11:03:35 searx docker[80054]: searxng  |     response = self.handle_exception(e)
Jul 04 11:03:35 searx docker[80054]: searxng  |                ^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/venv/lib/python3.12/site-packages/flask/app.py", line 1511, in wsgi_app
Jul 04 11:03:35 searx docker[80054]: searxng  |     response = self.full_dispatch_request()
Jul 04 11:03:35 searx docker[80054]: searxng  |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/venv/lib/python3.12/site-packages/flask/app.py", line 919, in full_dispatch_request
Jul 04 11:03:35 searx docker[80054]: searxng  |     rv = self.handle_user_exception(e)
Jul 04 11:03:35 searx docker[80054]: searxng  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/venv/lib/python3.12/site-packages/flask/app.py", line 915, in full_dispatch_request
Jul 04 11:03:35 searx docker[80054]: searxng  |     rv = self.preprocess_request()
Jul 04 11:03:35 searx docker[80054]: searxng  |          ^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/venv/lib/python3.12/site-packages/flask/app.py", line 1291, in preprocess_request
Jul 04 11:03:35 searx docker[80054]: searxng  |     rv = self.ensure_sync(before_func)()
Jul 04 11:03:35 searx docker[80054]: searxng  |          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/searx/webapp.py", line 435, in pre_request
Jul 04 11:03:35 searx docker[80054]: searxng  |     preferences = Preferences(themes, list(categories.keys()), engines, searx.plugins.STORAGE, client_pref)
Jul 04 11:03:35 searx docker[80054]: searxng  |                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  |   File "/usr/local/searxng/searx/preferences.py", line 415, in __init__
Jul 04 11:03:35 searx docker[80054]: searxng  |     choices=list(favicons.proxy.CFG.resolver_map.keys()) + ['']
Jul 04 11:03:35 searx docker[80054]: searxng  |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Jul 04 11:03:35 searx docker[80054]: searxng  | AttributeError: 'NoneType' object has no attribute 'resolver_map'
```

--- a/searx/webapp.py
+++ b/searx/webapp.py
@@ -1350,9 +1350,9 @@ def is_werkzeug_reload_active() -> bool:
     .. _werkzeug.serving:
        https://werkzeug.palletsprojects.com/en/stable/serving/#werkzeug.serving.run_simple
     """
-
-    if "uwsgi" in sys.argv:
-        # server was launched by uWSGI
+    logger.debug("sys.argv: %s", sys.argv)
+    if "uwsgi" in sys.argv[0] or "granian" in sys.argv[0]:
+        # server was launched by granian (or uWSGI)
         return False
 
     # https://github.com/searxng/searxng/pull/1656#issuecomment-1214198941

